tests/Api.IntegrationTests/Endpoints/ImportNotifications/GetUpdatedTests.cs

```diff
@@ -33,19 +35,30 @@ public async Task Get_ShouldSucceed()
                 }
             );
 
-        var response = await client.GetStringAsync(Testing.Endpoints.ImportNotifications.GetUpdated(bcp));
+        var from = DateTime.Now.Subtract(TimeSpan.FromHours(1));
+        var to = DateTime.Now.Subtract(TimeSpan.FromMinutes(30));
 
+        var url = Testing.Endpoints.ImportNotifications.GetUpdated(from, to, bcp);
+
+        var response = await client.GetStringAsync(url);
         await VerifyJson(response).UseStrictJson().DontScrubGuids().DontScrubDateTimes();
     }
 
-    [Fact(Skip = "Not implemented yet")]
-    public async Task Get_NoBcp_ShouldFail()
+    [Fact]
+    public async Task Get_ShouldFail_With400_BadRequest()
```

Comment:
You got rid of my test :(

Reply:
I'm not sure it would have actually failed. We may need an "at least one" attribute or something to enforce at least one BCP being submitted

Reply:
Most likely just a check on BCP quantity within the validator itself